index.html

```diff
@@ -225,6 +227,7 @@ <h3 class="partners-desktop-title">Partners</h3>
             <hr class="about-desktop-hr">
             <ul class="partners-ul">
                 <img class="partners-img" src="img/ethereum.svg" alt="">
+                <img class="partners-img" src="img/chainlink.svg" alt="">
                 <img class="partners-img" src="img/synthetix.svg" alt="">
                 <img class="partners-img" src="img/storj.svg" alt="">
                 <img class="partners-solana-img" src="img/solana.svg" alt="">
```

Comment:
- [ ] Kindly make sure all `alt` text is not empty in all your `.html` files in order to increase page accessibility. :wink: